book/analysis/big-o-notation.md

```diff
@@ -5,104 +5,106 @@ position: 2
-insignificant as *n* gets large. We would say then that the function
-$$T(n)$$ has an order of magnitude $$f(n)=n^{2}$$, or simply that it is
-$$O(n^{2})$$.
-
-Although we do not see this in the summation example, sometimes the
-performance of an algorithm depends on the exact values of the data
-rather than simply the size of the problem. For these kinds of
-algorithms we need to characterize their performance in terms of best
-case, *worst case*, or *average case* performance. The worst case
-performance refers to a particular data set where the algorithm performs
-especially poorly. Whereas a different data set for the exact same
-algorithm might have extraordinarily good performance. However, in most
-cases the algorithm performs somewhere in between these two extremes
-(average case). It is important for a computer scientist to understand
-these distinctions so they are not misled by one particular case.
-
-A number of very common order of magnitude functions will come up over
-and over as you study algorithms. These are shown in
-the table below. In order to decide which of these
-functions is the dominant part of any $$T(n)$$ function, we must see how
-they compare with one another as *n* gets large.
+steps is $$T(n)=5n^{2}+27n+1005$$. When *n* is small (1 or 2), the constant 1005
+seems to be the dominant part of the function. However, as *n* gets larger, the
+$$n^{2}$$ term becomes the most important, dwarfing the other two terms in its
+significance to the final result.
+
+Again, for an *approximation* of $$T(n)$$ at large values of *n*, we can focus
+on $$5n^{2}$$ and ignore the other terms. Similarly, the coefficient $$5$$
+becomes insignificant as *n* gets larger. We would say then that the function
+$$T(n)$$ has an order of magnitude $$f(n)=n^{2}$$; more simply, the function
+$$T(n)$$ is $$O(n^{2})$$.
+
+Although we do not see this in the summation example, sometimes the performance
+of an algorithm depends on the **exact** data values rather than just the size
+of the problem. For these kinds of algorithms, we need to characterize their
+performances as *worst case*, *best case*, or *average case*.
+
+The worst case performance refers to a particular data set where the algorithm
+performs especially poorly, whereas a different data set might have amazing
+performance. In most cases, however, the algorithm performs somewhere in between
+these two extremes, in what we kindly call the average case. Understanding these
+distinctions can help prevent any one particular case from misleading us.
+
+There are several common order of magnitude functions that will crop up
+repeatedly as you study algorithms. These are shown in the table below. In order
+to decide which of these functions is the dominant part of a $$T(n)$$ function,
+we must see how it compares with the others as $$n$$ becomes larger.
 
 f(n) | Name
 --- | ---
 $$1$$ | Constant
-$$log n$$ | Logarithmic
+$$log{\cdot}n$$ | Logarithmic
```

Comment:
@aleksharma12 we don't want the dot here... log is a function over n. If you think it's confusing we can do `$$log(n)$$`.